Automattic/vip-go-mu-plugins
# Add/jetpack 10.6 subtree

## Description
Adds the Jetpack 10.6 tag ref as a subtree.

## Changelog Description
###  Jetpack 10.6 Added
VIP users can pin their Jetpack version to 10.6

## Checklist

## Steps to Test
1. Check out PR.
1. Pin Jetpack to 10.6
1. Go to `wp-admin` > `Jetpack`
1. Verify Jetpack version is 10.6
